static/js/user_groups_settings_ui.js

```diff
@@ -133,6 +133,28 @@ export function add_group_to_table(group) {
     }
 }
 
+export function update_group(group_id) {
```

Comment:
This probably needs a check for whether the settings pane is open, and to do nothing if it isn't.

Reply:
Ahh, you do that in the caller; I think it might be cleaner to do that here for readability reasons, or have a comment at the top here saying that the caller is responsible for checking the user group settings panel is open..

Reply:
Have moved the check here.